.github/workflows/integration-test-flex-suite.yaml

```diff
@@ -14,10 +18,65 @@ jobs:
         go-version: 1.15
 
     - name: setup minikube
-      uses: manusa/actions-setup-minikube@v2.2.0
+      uses: manusa/actions-setup-minikube@v2.3.0
       with:
-        minikube version: 'v1.13.1'
-        kubernetes version: 'v1.19.2'
+        minikube version: 'v1.16.0'
+        kubernetes version: ${{ matrix.kubernetes-versions }}
+        start args: --memory 6g --cpus=2
+
+    - name: check k8s cluster status
+      run: |
+        kubectl cluster-info
+        kubectl get pods -n kube-system
+
+    - name: use local disk
+      run: |
+        sudo swapoff --all --verbose
+        sudo umount /mnt
+        # search for the device since it keeps changing between sda and sdb
+        sudo wipefs --all --force /dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1
+        sudo lsblk
+
+    - name: build rook
+      run: |
+        # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
+        GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
+        docker images
+        docker tag $(docker images|awk '/build-/ {print $1}') rook/ceph:master
+
+    - name: TestCephFlexSuite
+      run: |
+       export DEVICE_FILTER=$(lsblk|awk '/14G/ {print $1}'| head -1)
+       go test -v -timeout 1800s -run CephFlexSuite github.com/rook/rook/tests/integration
+
+    - name: Artifact
+      uses: actions/upload-artifact@v2
+      if: failure()
+      with:
+        name: ceph-flex-suite-artifact
+        path: /home/runner/work/rook/rook/tests/integration/_output/tests/
+
+  TestCephFlexSuite-Master:
+    if:  github.event_name != 'pull_request'
```

Comment:
What's the purpose of this condition?

Reply:
to trigger ci this in case of the master branch

Reply:
Hum but the `-master`all got skipped, don't we want the opposite?

Reply:
changed.